<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Scribo Demo - v0.1</string>
  </property>
  <property name="dockOptions" >
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" name="verticalLayout_2" >
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <layout class="QGridLayout" name="gridLayout" >
        <item row="0" column="0" colspan="4" >
         <widget class="QLabel" name="description" >
          <property name="text" >
           <string>Write description here.</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLineEdit" name="filepath" />
        </item>
        <item row="1" column="1" >
         <widget class="QPushButton" name="browseBtn" >
          <property name="text" >
           <string>&amp;Browse</string>
          </property>
          <property name="icon" >
           <iconset resource="../demo.qrc" >
            <normaloff>:/icons/document-open.png</normaloff>:/icons/document-open.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="2" >
         <widget class="QPushButton" name="displayBtn" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>Display</string>
          </property>
          <property name="icon" >
           <iconset resource="../demo.qrc" >
            <normaloff>:/icons/view-refresh.png</normaloff>:/icons/view-refresh.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="3" >
         <widget class="QPushButton" name="runBtn" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>Run</string>
          </property>
          <property name="icon" >
           <iconset resource="../demo.qrc" >
            <normaloff>:/icons/go-next.png</normaloff>:/icons/go-next.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget" >
        <property name="currentIndex" >
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab" >
         <attribute name="title" >
          <string>Image</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3" >
          <item>
           <widget class="QGraphicsView" name="imageResult" />
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2" >
         <attribute name="title" >
          <string>Text</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4" >
          <item>
           <widget class="QTextBrowser" name="textBrowser" >
            <property name="html" >
             <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Test&lt;/p>&lt;/body>&lt;/html></string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>658</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
 </widget>
 <resources>
  <include location="../demo.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>filepath</sender>
   <signal>returnPressed()</signal>
   <receiver>displayBtn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>201</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel" >
     <x>516</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
